apps/backend/src/groups/groups.service.ts

```diff
@@ -0,0 +1,86 @@
+import { Injectable, NotFoundException } from '@nestjs/common';
+import { Group, GroupMembers, Prisma, Role, User } from '@prisma/client';
+import { PrismaService } from 'nestjs-prisma';
+
+@Injectable()
+export class GroupsService {
+  constructor(private readonly prisma: PrismaService) {}
+
+  async create(data: Prisma.GroupCreateInput): Promise<Group> {
+    return await this.prisma.group.create({ data });
```

Comment:
You can put these in try-catch blocks also, so we won't get error code 500 when i. e. we try to create a group with non existing members